container-image/examples/openshift/README.md

```diff
@@ -0,0 +1,35 @@
+# WildFly Chat Bot deployed to OpenShift
+
+You must be already logged into your OpenShift cluster.
+
+* Deploy your WildFly [helloworld quickstart](https://github.com/wildfly/quickstart/tree/main/helloworld) application:
+
+`helm install wildfly-app -f helm-wildfly.yaml wildfly/wildfly`
+
+* Patch the deployed application service (generated by helm chart) to expose the management port (used by the WildFly chat bot):
+
+```
+oc patch service/wildfly-app -p --type='json' '[{"op": "replace", "path": "/spec/ports/0/name", "value":"http"}]'
+
+oc patch service/wildfly-app -p '{"spec": {"ports": [{"port": 9990, "targetPort": "admin", "protocol": "TCP", "name": "admin"}]}}'
```

Comment:
maybe, we should do something directly in the Helm Chart for WildFly to expose the admin port.